### DISCUSSION TITLE
Is possible to add a component that lets you switch between different themes?

### DISCUSSION THREAD

**Discussion by Jayrr-Dev** [skeletonlabs/skeleton#1798] [Ideas]
I really like the drop down in the the skeleton website where it lets you switch between different themes. But, disappointed to find that it wasn't a standard component in Skeleton. It would be really awesome to have on a website. 

Just my thoughts on it.

**Comment by endigo9740** [comment]
The last question on this board was asking about this and was answered:

- https://github.com/skeletonlabs/skeleton/discussions/1796
